### ISSUE TITLE
A problem about loss function

### ISSUE THREAD

**Issue by bosaiiasliu** [Nebularaid2000/bottleneck#1]
I want to know if the negative sign of the negative entropy loss in trainer_dropout_entropy_deltav_baseline.py is taken one more time? It looks like there are negative signs on both lines 283 and 234. If yes, is this a bug? My experiments using rate1=0.7, rate2=1, the loss that penalizes higher-order interactions, show that models trained in this way exhibit higher-order interactions that are encouraged rather than penalized.

**Comment by bosaiiasliu** [comment]
Sorry, there are no bugs. I made a mistake.
